cookbook/doctrine/reverse_engineering.rst

```diff
@@ -101,8 +101,10 @@ execute the second command only.
 
 .. tip::
 
-    If you want to use annotations, you can safely delete the XML (or YAML) files
-    after running these two commands.
+    If you want to use annotations, you must remove the XML (or YAML) files
+    after running these two commands. It is `not possible to mix XML/YAML metadata
+    definitions with annotated PHP entity class definitions 
+    <http://symfony.com/doc/current/book/doctrine.html#add-mapping-information>`_.
```

Comment:
The addition is nice, but we should use the `ref` role to link to that section like this (I would also reword this a bit):

```rst
If you want to use annotations, you must remove the XML (or YAML) files
after running these two commands. This is necessary as
:ref:`it is not possible to mix mapping configuration formats <book-doctrine-adding-mapping>`.